Ginger/GingerCoreCommon/UIElement/ElementInfo.cs

```diff
@@ -289,6 +297,8 @@ public enum eLocateBy
         ByModelName,
         [EnumValueDescription("By CSS Selector")]
         ByCSSSelector,
+        [EnumValueDescription("POM Element")]
+        POMElement,
```

Comment:
please set the Description as "Page Objects Model Element" (because we not supposed to use POM on UI, only in our backend)